src/renderer/components/MainPage.tsx

```diff
@@ -274,12 +238,17 @@ export default class MainPage extends React.PureComponent<Props, State> {
         if (addedIndex === undefined || removedIndex === addedIndex) {
             return;
         }
-        const teamIndex = this.props.moveTabs(removedIndex, addedIndex < this.props.teams.length ? addedIndex : this.props.teams.length - 1);
+        const currentTabs = this.props.teams.find((team) => team.name === this.state.activeServerName)?.tabs;
+        if (!currentTabs) {
+            // TODO: figure out something here
```

Comment:
isn't messaging always displayed?

Reply:
This is just a null check in case a team isn't found for some reason.